server/src/main/java/org/candlepin/controller/ContentManager.java

```diff
@@ -324,12 +324,15 @@ public Content updateContent(ContentData update, Owner owner, boolean regenerate
 
         if (contentData == null || contentData.isEmpty()) {
             // Nothing to import
-            return new HashMap<String, Content>();
+            new ImportResult<Content>(Collections.<String, Content>emptyMap(),
+                Collections.<String, Content>emptyMap(), Collections.<String, Content>emptyMap());
         }
 
-        Map<String, Content> importedContent = new HashMap<String, Content>();
+        Map<String, Content> skippedContent = new HashMap<String, Content>();
         Map<String, Content> createdContent = new HashMap<String, Content>();
         Map<String, Content> updatedContent = new HashMap<String, Content>();
+        ImportResult<Content> importResult = new ImportResult<Content>(skippedContent, createdContent,
```

Comment:
ImportResult is created here and used just to be returned at the end of the method. What about construct it just at line "return importResult"?